optimize/self-managed/optimize-deployment/configuration/webhooks.md

```diff
@@ -1,67 +0,0 @@
----
-id: webhooks
-title: "Webhooks"
-description: "Read about how to configure alert notification webhooks for alerts on custom systems."
----
-
-<span class="badge badge--platform">Camunda 7 only</span>
```

Comment:
There is a page called "Common problems". I don't think these are a good fit for C8 either, can we delete this too? https://preview.docs.camunda.cloud/pr-4785/optimize/next/self-managed/optimize-deployment/configuration/common-problems/